app/controllers/heartbeat_controller.rb

```diff
@@ -2,12 +2,20 @@ class HeartbeatController < ApplicationController
   permit_only_with_key
 
   def healthcheck
-    render json: {
-      checks: {
-        sendgrid: sendgrid_alive?,
-        messagelabs: messagelabs_alive?,
-        database: ActiveRecord::Base.connection.active?
-      }
+    database_active = begin
+      ActiveRecord::Base.connection.active?
+    rescue PG::ConnectionBad
```

Comment:
It's fine. If anything we'll get rid of the database completely when we integrate with the API, so no need to worry about it IMO.